kubernetes/kuma-demo-aio.yaml

```diff
@@ -136,12 +136,14 @@ spec:
       app: kuma-demo-backend
       version: v0
       env: prod
+      protocol: http
```

Comment:
This is not how we describe that service runs on HTTP.
https://kuma.io/docs/0.4.0/policies/http-support-in-kuma/